R/parse-news-items.R

```diff
@@ -231,12 +231,7 @@ add_squash_info <- function(description) {
 parse_merge_commit <- function(message) {
   pr_data <- harvest_pr_data(message)
   pr_number <- pr_data$pr_number
-  pr_numbers <- toString(
-    sprintf(
-      "#%s",
-      c(unlist(pr_data$issue_numbers), pr_number)
-    )
-  )
+  pr_numbers <- toString(c(unlist(pr_data$issue_numbers), paste0("#", pr_number)))
```

Comment:
At some point, this could also be some {cli} function that takes care of the Oxford comma.